### ISSUE TITLE
Builder: Popup message on installation

### ISSUE THREAD

**Issue by webiummedia** [quasar/QuasarRAT#386]
In the builder when the client is installed successfully, we could add a feature where you can choose an icon (Red cross, Yellow exclamation mark, dummy user prompt [...]) and compose a popup message.

**Comment by yankejustin** [comment]
Not sure what you mean. Are you speaking of the event of correct client installation or the builder form where a client is correctly built?

**Comment by webiummedia** [comment]
It would be a box that would appear client side upon installation to confirm that it was successful. Cause right now you click and nothing happens. The message and icon should be modifiable in case the rat admin binded it with something else to stealth it. In my case, I binded it with a firefox installation package. I would of put something like "You are now ready to install Firefox". That would of been my queue that the rat was installed.

**Comment by yankejustin** [comment]
Oh, I see. That sounds like a good idea.

**Comment by rabbitsmith** [comment]
I think this can be added as a check box so that this can be optional during the building of client

**Comment by webiummedia** [comment]
Yes first a checkbox then a title and message field with a choice of icon selection (red cross, yellow exclamation, blue exclamation, help bubble) the standard alert box stuff.
